test/module/addon/CMakeLists.txt

```diff
@@ -31,7 +31,13 @@ set_target_properties(${TARGET_NAME} PROPERTIES
 target_include_directories(${TARGET_NAME}
     PRIVATE
     ${NAPA_ROOT}/inc
-    "/usr/local/include/node")
+    ${CMAKE_JS_INC})
```

Comment:
please pay attention to simple-addon npm test cases. 2 of them run to failure in my client.